Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/GenerateKeyFlow.swift

```diff
@@ -139,6 +139,10 @@ class GenerateKeyFlow: AddKeyFlow {
         }
         show(keyVC)
     }
+
+    override func didKeyAdded() {
```

Comment:
@elgatovital it should also work for Import Key flow

Reply:
I don't think backup should be started for imported keys (they are already persisted somewhere outside). Everything else works as before

Reply:
oh, that's a backup! True, I mis-recognized it as a 'delegate key'.